DemoServer/Controllers/Demos/StaticIndexes/FanoutIndex/metadata.json

```diff
@@ -0,0 +1,159 @@
+{
+    "Slug": "fanout-index",
+    "SourceFileName": "FanoutIndexController.cs",
+    "Title": "Fanout Index",
+    "DescriptionLines": [
+        "When a document field is an array, the inner array fields can be indexed in what is called a **Fanout Index.**",
+        "",
+        "An index entry will be generated per item of the array,",
```

Comment:
And after that, we can explain that one of such cases is processing of array properties, where each element of an array will result in one index entries

Reply:
see above comment